### PR TITLE
Updated cabal file metadata

### PR DESCRIPTION
**Pull request by Soupstraw**
This PR updates some out-of date metadata in cabal files. I could not come up with good descriptions/synopses for the packages.

#2999

### CODE REVIEW

**Comment by Soupstraw** [comment]
~I'll also do #2998, since I'm already changing cabal files anyways.~
Never mind, @teodanciu is already working on it.

File: eras/alonzo/impl/cardano-ledger-alonzo.cabal [hunk 1]
```diff
@@ -8,9 +8,8 @@ description:
   via Plutus Core.
 bug-reports:         https://github.com/input-output-hk/cardano-ledger/issues
 license:             Apache-2.0
-author:              IOHK Formal Methods Team
-maintainer:          formal.methods@iohk.io
-copyright:           2020 Input Output (Hong Kong) Ltd.
```
Comment (teodanciu) [inline eras/alonzo/impl/cardano-ledger-alonzo.cabal:13]: I wonder if Jared's [comment](https://github.com/input-output-hk/cardano-ledger/issues/2999#issuecomment-1227475577)   
 > be consistent with the packages in the cardano-node repository with respect to the maintainer email and author 

implies that we should get rid of the copyright, or it refers only to these two fields :thinking:

Comment (Soupstraw) [reply eras/alonzo/impl/cardano-ledger-alonzo.cabal:13]: Other node packages don't have the `copyright` field, so Jared suggested we remove it.

Comment (teodanciu) [reply eras/alonzo/impl/cardano-ledger-alonzo.cabal:13]: Makes sense :+1:

File: libs/cardano-ledger-test/cardano-ledger-test.cabal [hunk 2]
```diff
@@ -4,13 +4,9 @@ name:                cardano-ledger-test
 version:             0.1.0.0
 synopsis:
   Testing harness, tests and benchmarks for Shelley style cardano ledgers
--- description:
--- bug-reports:
 license:             Apache-2.0
-author:              IOHK Formal Methods Team
+author:              IOHK
 maintainer:          nicholas.clarke@iohk.io
```
Comment (teodanciu) [inline libs/cardano-ledger-test/cardano-ledger-test.cabal]: Did you miss this one?

Comment (Soupstraw) [reply libs/cardano-ledger-test/cardano-ledger-test.cabal]: Nice catch

**Review by teodanciu — APPROVED** [review]
Looks good to me. I don't know if the changelog update is required, since this change doesn't impact anyone, but I don't understand exactly who the audience of the changelog is, so I can't have an opinion on it

File: CHANGELOG.md [hunk 3]
```diff
@@ -51,6 +50,7 @@ in the naming of release branches.
   - `addrShelleyWitsL` to `addrShelleyTxWitsL`
   - `bootAddrShelleyWitsL` to `bootAddrShelleyTxWitsL`
   - `scriptShelleyWitsL` to `scriptShelleyTxWitsL`
+- Updated package metadata #3023
```
Comment (lehins) [inline CHANGELOG.md:54]: I don't think this is needed, changelog usually only reflects code changes that can affect downstream users.

```suggestion
```

**Review by lehins — APPROVED** [review]
As @teodanciu said, changelog entry is not needed, otherwise it looks good.

**Review by JaredCorduan — APPROVED** [review]
Looks great, thank you!

**Pull request merged**